frameworks/cassandra/tests/test_recovery.py

```diff
@@ -27,6 +27,17 @@ def configure_package(configure_universe):
 
 @pytest.mark.sanity
 @sdk_utils.dcos_1_9_or_higher # dcos task exec not supported < 1.9
+def test_node_replace_replaces_seed_node():
```

Comment:
Is the ring membership part of the recovery success check? I still think that the flakiness is better handled by the improved recovery check in #1369, but I'm not against changing the order of the tests. Another issue could be the fact that we set one of the nodes to unusable in the other test, and that makes this deployment more tricky (also solved by switching the order).